lib/gateway/Shard.js

```diff
@@ -510,17 +515,17 @@ class Shard extends EventEmitter {
             }
             case "MESSAGE_DELETE": {
                 const channel = this.client.getChannel(packet.d.channel_id);
-                if(!channel) {
-                    break;
-                }
+
                 /**
                 * Fired when a cached message is deleted
                 * @event Client#messageDelete
-                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. No other property is guaranteed
+                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. No other property is guaranteed. If the channel is not cached, channel will be an object with only `id`.
```

Comment:
consistency
```suggestion
                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. No other property is guaranteed. If the channel is not cached, channel will be an object with an `id` key.
```